src/panels/config/repairs/ha-config-repairs.ts

```diff
@@ -52,6 +52,7 @@ class HaConfigRepairs extends LitElement {
               @click=${this._openShowMoreDialog}
             >
               <img
+                alt=${domainToName(this.hass.localize, issue.domain)}
```

Comment:
There may be a point of discussion here. The code originally just set a `title` property on the node. While this is good for non-a11y cases, the `title` property is not necessarily read by assistive technologies. Only the `alt` attribute is guaranteed to be considered. Therefore, I set both. Some screen readers (i.e., NVDA) will read both fields, but this is user configurable. As I understand it, JAWS will just not read the `title` tag at all.

Reply:
The `alt` attribute takes precedence per the accessible name computation spec and will certainly be used by all user agents so this is the correct markup.  The `title` is just there for the browser tooltip.